jflex/src/main/java/jflex/core/IntCharSet.java

```diff
@@ -185,43 +190,7 @@ public void add(Interval interval) {
    * @param c Character to add.
    */
   public void add(int c) {
```

Comment:
This was a specifically optimised version for a single character with simpler logic for merging intervals. I'd like to keep that. Can you please revert this bit?

Reply:
Actually, don't worry, I have another commit incoming and will put it into that one.